drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -54,10 +54,12 @@ local ZOOZ_ZEN_30_DIMMER_RELAY_FINGERPRINTS = {
 
 local function version_report_handler(self, device, cmd)
   if cmd.args.firmware_0_version > 1 or
-  (cmd.args.firmware_0_version == 1 and cmd.args.firmware_0_sub_version > 4) then
+    (cmd.args.firmware_0_version == 1 and cmd.args.firmware_0_sub_version > 4) then
```

Comment:
what if firmware_0_version is > 1 and firmware_0_sub_version <=4 ?
You might add another conditions, like:
`if ((cmd.args.firmware_0_version > 1) or (cmd.args.firmware_0_version == 1 and cmd.args.firmware_0_sub_version > 4)) then`